apps/coursition/src/app/api/payments/route.ts

```diff
@@ -3,6 +3,13 @@ import { type NextRequest, NextResponse } from 'next/server'
 
 export const POST = async (request: NextRequest) => {
   const rawBody = await request.text()
-  const data = await webhookEventHandler(rawBody, request)
+  const data = await webhookEventHandler({
+    rawBody,
+    request,
+    customData: {
+      callback: () => void console.log('user granted lifetime access'),
```

Comment:
I think we should check that `product_name` is `Lifetime` as specified here:
https://docs.lemonsqueezy.com/guides/developer-guide/webhooks#example-webhook-data

I already created package called "Lifetime", so it should be easier to handle like this.